tests/ui/unnecessary_vec_drain.stderr

```diff
@@ -0,0 +1,16 @@
+error: unnecessary iterator `Drain` is created and dropped immedietly
+  --> $DIR/unnecessary_vec_drain.rs:7:5
+   |
+LL |     vec.drain(..);
+   |     ^^^^^^^^^^^^^ help: consider calling `clear()`: `vec.clear()`
+   |
+   = note: `-D clippy::unnecessary-vec-drain` implied by `-D warnings`
+
+error: unnecessary iterator `Drain` is created and dropped immediately
+  --> $DIR/unnecessary_vec_drain.rs:8:9
+   |
+LL |     vec.drain(0..vec.len());
+   |         ^^^^^^^^^^^^^^^^^^^ help: consider calling `clear()`: `vec.clear()`
```

Comment:
This suggestion won't work. The code after applying this suggestion would be `vec.vec.clear()`. You'll have to fix the span your using when producing this suggestion to also include `vec`.